home/src/main/resources/config/default.runtime.properties

```diff
@@ -28,8 +28,8 @@ vitro.local.solr.url = http://localhost:8983/solr/vitrocore
 #     email.smtpHost = smtp.mydomain.edu
 #     email.replyTo = vitroAdmin@mydomain.edu
 #
-email.smtpHost =
-email.replyTo =
+# email.smtpHost =
```

Comment:
We should align: https://github.com/vivo-project/VIVO/pull/222/commits/1ca40cd7b0703ac60fe938cf37a9b16c8501d2c7

Reply:
Getting closer to alignment... if you remove the ['Example' line](https://github.com/vivo-project/VIVO/pull/222/files#diff-863e26484eafb9690e1a9e118277d6e69045d655a07f0587b22ffacead7cbb2cR54) we will be there. I was following the format for the Solr variable below 🤷

Reply:
:link: